src/protocols/radius/encode.c

```diff
@@ -724,7 +772,7 @@ static ssize_t attr_shift(uint8_t const *start, uint8_t const *end,
 	 */
 
 	while (check_len > (255 - hdr_len)) {
-		total += hdr_len;
+		new_hdr_count++;
```

Comment:
I'm not sure why `total` was changed to `new_hdr_count` and then a later multiplication.  It should just use `total` as before

Reply:
Agreed; I will change it back.